clients/drcachesim/simulator/caching_device.cpp

```diff
@@ -112,9 +112,9 @@ caching_device_t::request(const memref_t &memref_in)
         // Make sure last_tag is properly in sync.
         assert(tag != TAG_INVALID &&
                tag == get_caching_device_block(last_block_idx, last_way).tag);
-        stats->access(memref_in, true /*hit*/);
+        stats->access(memref_in, true /*hit*/, TAG_INVALID);
```

Comment:
(So for hit==true we always pass TAG_INVALID but I suppose the reverse is not true and we can't get rid of the bool param and infer a hit from TAG_INVALID -- we'd need a different sentinel.)

Reply:
Changed the parameter type to a pointer to the cache block. Since we are passing a pointer to the cache block for both hits and misses, I decided to keep this boolean flag.